apps/android/app/src/main/res/values/strings.xml

```diff
@@ -58,18 +58,39 @@
   <string name="it_can_disabled_via_settings_notifications_still_shown"><b>It can be disabled via settings</b> – notifications will still be shown while the app is running.</string>
   <string name="turn_off_battery_optimization">In order to use it, please <b>disable battery optimization</b> for <xliff:g id="appName">SimpleX</xliff:g> in the next dialog. Otherwise, the notifications will be disabled.</string>
-  <string name="turning_off_background_service">Battery optimization is active, turning off Private notifications. You can re-enable them via settings.</string>
+  <string name="turning_off_service_and_periodic">Battery optimization is active, turning off background service and periodic requests for new messages. You can re-enable them via settings.</string>
+  <string name="periodic_notifications">Periodic notifications</string>
+  <string name="periodic_notifications_disabled">Periodic notifications are disabled!</string>
+  <string name="periodic_notifications_desc">To preserve your privacy, instead of push notifications the app can fetch new messages periodically – it uses a few percent of the battery per day.</string>
 
   <!-- SimpleX Chat foreground Service -->
   <string name="simplex_service_notification_title"><xliff:g id="appNameFull">SimpleX Chat</xliff:g> service</string>
   <string name="simplex_service_notification_text">Receiving messages…</string>
   <string name="hide_notification">Hide</string>
 
+  <!-- Notifications -->
+  <string name="settings_notifications_mode_title">Background service</string>
+  <string name="settings_notification_preview_mode_title">Show preview</string>
+  <string name="notifications_mode_off">Starts only in foreground</string>
+  <string name="notifications_mode_periodic">Starts periodically</string>
+  <string name="notifications_mode_service">Always on</string>
+  <string name="notifications_mode_off_desc">App can receive notifications only when it\'s running, no background service will be started</string>
+  <string name="notifications_mode_periodic_desc">Checks new messages every 10 minutes for about 10 seconds, then stops</string>
+  <string name="notifications_mode_service_desc">Background service is always on. Means every new message will be shown as fast as possible</string>
+  <string name="notification_preview_mode_message">Message text</string>
+  <string name="notification_preview_mode_contact">Contact name</string>
+  <string name="notification_preview_mode_hidden">Hidden</string>
+  <string name="notification_preview_mode_message_desc">Show an author and a message</string>
+  <string name="notification_preview_mode_contact_desc">Show only an author without message</string>
+  <string name="notification_display_mode_hidden_desc">Hide an author and a message, show only the presence of the message</string>
```

Comment:
```suggestion
  <string name="notification_preview_mode_message_desc">Show the contact and the message</string>
  <string name="notification_preview_mode_contact_desc">Show only the contact</string>
  <string name="notification_display_mode_hidden_desc">Hide the contact and the message</string>
```

Reply:
I'd remove "the" so it reads easier - just "show contact and message", "show only contact", etc. even if it's grammatically incorrect. That footer is not the most official place in the app, and the footer's font is quite small so the less letters to read the better